components/item-info.js

```diff
@@ -144,6 +145,7 @@ export default function ItemInfo ({
           </Link>}
         {me && !item.meSats && !item.position &&
           !item.mine && !item.deletedAt && <DontLikeThisDropdownItem id={item.id} />}
+        {me && !item.pollCost && !item.url && !item.bounty && <CrosspostDropdownItem item={item} />}
```

Comment:
Since this is about sharing data with nostr, I think it would make more sense to put this into the dropdown of the share button.

However, this would mean that you can only crosspost something that is selected as the root item since only then the share button shows up. But I think that's not a problem.

![2023-11-23-063359_1920x1080_scrot](https://github.com/stackernews/stacker.news/assets/27162016/d273800b-763a-4e07-87f4-31d37ce1aea8)

![2023-11-23-063408_1920x1080_scrot](https://github.com/stackernews/stacker.news/assets/27162016/0f94a611-87df-4c5d-8058-132d5cc27e16)

But then the `nostr note` link would also show up there which is no longer necessarily about "sharing data" :thinking: 

 What do you think?